packages/library-export-default-webpack-plugin/package.json

```diff
@@ -33,7 +33,7 @@
 		"webpack-sources": "^1.1.0"
 	},
 	"peerDependencies": {
-		"webpack": "^4.0.0"
+		"webpack": "^4.0.0 || ^5.0.0"
```

Comment:
I'm surprised it works with webpack 5. It's the hackiest plugin we created to this date 😅

Reply:
It's a very simple one though 🙂 

The plugin may be not needed at all: webpack already has an `output: { libraryExport: 'default' }` option that does the same thing. And the implementation is very similar to yours -- just append `.default` at the end 🙂.

The only caveat is that we want to export the `default` only for certain libraries/entrypoints, while the option is global 🙁